app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -253,6 +253,15 @@ class Settings private constructor(
         default = true
     )
 
+    var shouldShowSearchSuggestionsInPrivate by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_show_search_suggestions_in_private),
+        default = false
+    )
+
+    var showSearchSuggestionsInPrivateOnboardingFinished by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_show_search_suggestions_in_private_onboarding),
+        default = false)
```

Comment:
nit: Change the parenthesis location to match with other preferences
```suggestion
        default = false
    )
```